x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/plugin/MatchOperatorIT.java

```diff
@@ -128,6 +129,48 @@ public void testNotWhereMatch() {
         }
     }
 
+    public void testWhereMatchWithScoring() {
+        assumeTrue("'METADATA _score' is disabled", EsqlCapabilities.Cap.METADATA_SCORE.isEnabled());
+        var query = """
+            FROM test
+            METADATA _score
+            | WHERE content MATCH "fox"
+            | KEEP id, _score
```

Comment:
Do we allow `_score` to be used in the other processing commands in ES|QL? For example, eval/where/stats/sort/drop/rename/dissect/grok/enrich/mvexpand etc, there are some commands under snapshots. It will be great to have tests to capture the current behavior when `_score` is used there.

Reply:
good point, I'll add tests to cover this aspect